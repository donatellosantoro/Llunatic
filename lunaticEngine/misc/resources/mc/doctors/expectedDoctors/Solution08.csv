#doctor
npi,name,hospital,email
5555,G. House,PPTH,house@ppth.com
5556,_L161,NULL,NULL
1112,R. Chase,Sacred Hearth,chase@sh.com
1222,C. Taub,NULL,NULL

#prescription
rxid,patient,npiref
P1,J. Doe,1222
P3,M. White,5556
P2,W. Smith,5555
_L10,J. Doe,1112

