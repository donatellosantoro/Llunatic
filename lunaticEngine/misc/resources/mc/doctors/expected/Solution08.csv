#prescription
rxid,patient,npiref
P3,M. White,_L17
P1,J. Doe,_L447
P2,W. Smith,_L17

#doctor
npi,name,hospital,email
_L447,_L451,Sacred Hearth,chase@sh.com
_L17,_SKt0-N1(_L17),_SKt0-N2(_L17),_SKt0-N3(_L17)
_L423,G. House,PPTH,house@ppth.com
